examples/placegoose/src/db/utils.ts

```diff
@@ -0,0 +1,42 @@
+import fs from "node:fs";
```

Comment:
nit: so one of the reason why these db seeding utils (as well as the seed.ts file) lived outside of the main `src` folder, was that they relied on filesystem access, which cloudflare workers never have. 

it's fine to put them in here, but might be good to underscore with a comment that they're not really part of the Worker application, and they're more of "scripts"

Reply:
moved to a _scripts dir at root. i feel like it's good to create a boundary